src/components/TaskAnalysisTable/TaskAnalysisTable.jsx

```diff
@@ -515,7 +516,14 @@ const setupColumnTypes = (props, taskBaseRoute, manager, data, openComments) =>
         className="row-user-column"
         style={{color: AsColoredHashable(_get(row._original.completedBy, 'username') || row._original.completedBy).hashColor}}
       >
-        {_get(row._original.completedBy, 'username') || row._original.completedBy}
+        <a
+          className="mr-mx-4"
+          href={props.targetUserOSMProfileUrl()}
+          target='_blank'
+          rel="noopener noreferrer"
```

Comment:
Minor, but I think be better to just use `rel=noopener` here. It's probably useful to the openstreetmap.org staff to know how many user profile visits are coming from MapRoulette links (if we ask nicely they might even share that information back with us).